### PR TITLE
Use AdvanceTime action for initialization

### PR DESCRIPTION
**Pull request by wthrowe**
### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp [hunk 1]
```diff
@@ -274,16 +274,9 @@ struct InitializeElement {
       using Vars = typename variables_tag::type;
```
Comment (nilsdeppe) [inline src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp:274]: I'm confused by your commit message "set bot tags". What are "bot" tags?

Comment (kidder) [reply src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp:274]: I'm guessing a typo for "both"

**Review by kidder — COMMENTED** [review]
I'm happy once the typo in the commit message is fixed

**Comment by nilsdeppe** [comment]
Me too :)

**Comment by wthrowe** [comment]
@kidder has guessed correctly.  Fixed.

**Review by kidder — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Comment by nilsdeppe** [comment]
travis failures are timeouts.

**Pull request merged**